roles/setup_hammerdb/defaults/main.yml

```diff
@@ -34,4 +34,15 @@ supported_os:
   - Debian10
   - OracleLinux8
   - Ubuntu20
+
 use_patroni: false
+
+supported_hammerdb_versions:
```

Comment:
Let's leave this for 4.6 and 4.7 for now.  Since I believe those are the only two we have used thus far.

Reply:
updated!